ui/src/Products/ProductListGrouped.tsx

```diff
@@ -87,6 +89,7 @@ function GroupedByList({
                 });
             }
         }
+        setProductsGrouped(true);
```

Comment:
Not thrilled with this as a long-term solution, but I'm glad we have at least something in the short term

Reply:
Wouldn't it have to rerender already to update the list?